csrankings-c.csv

```diff
@@ -1264,6 +1265,7 @@ Clifford A. Shaffer,Virginia Tech,http://www.cs.vt.edu/~shaffer,v0tJmyYAAAAJ
 Clifford De Raffaele,Middlesex University,https://www.mdx.ac.uk/about-us/our-people/staff-directory/profile/de-raffaele-clifford,YcKa73MAAAAJ
 Clifford Stein 0001,Columbia University,http://www.columbia.edu/~cs2035,r49_E2cAAAAJ
 Clifford Sule,Middlesex University,https://www.mdx.ac.uk/about-us/our-people/staff-directory/profile/sule-clifford,WeHUq4YAAAAJ
+Clint Heyer,Malmö University,https://mau.se/en/persons/clint.heyer/,NOSCHOLARPAGE

```

Comment:
Not in CS. See https://mau.se/en/research/research-subjects/computer-science/#accordion-82295.